app/controllers/admin/community_memberships_controller.rb

```diff
@@ -91,6 +93,8 @@ def generate_csv_for(memberships)
             user.given_name,
```

Comment:
[This page](http://apidock.com/rails/ActiveRecord/Batches/ClassMethods/find_each) says that find_each ignores order_by. Should it really be used?